testsuite/capabilities.py

```diff
@@ -10,20 +10,20 @@
 @functools.cache
 def has_kuadrant():
     """Returns True, if Kuadrant deployment is present and should be used"""
-    if spokes := weakget(settings)["control_plane"]["spokes"] % {}:
-        for name, openshift in spokes.items():
+    project = settings["service_protection"]["system_project"]
+    if clusters := weakget(settings)["control_plane"]["additional_clusters"] % []:
+        for openshift in clusters:
+            name = openshift.api_url
             # Try if Kuadrant is deployed
             if not openshift.connected:
                 return False, f"Spoke {name} is not connected"
```

Comment:
"Spoke" is not the right terminology anymore, there are two occurrences in this file that would be nice to replace with something more accurate. Maybe just `Cluster` ?